address-controller/src/main/resources/swagger.json

```diff
@@ -0,0 +1,640 @@
+                "summary": "Delete an AddressSpace",
+                "description": "",
+                "operationId": "deleteAddressSpace",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "description": "Name of AddressSpace to delete",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Deletion successful; List of remaining AddressSpaces is returned.",
+                        "schema": {
+                            "$ref": "#/definitions/AddressSpaceList"
+                        }
+                    },
+                    "404": {
+                        "description": "AddressSpace not found"
+                    }
+                }
+            }
+        },
+        "/addresses/{addressSpace}": {
+            "get": {
+                "tags": [
+                    "addresses"
+                ],
+                "summary": "Get list of all Addresses in the AddressSpace",
+                "description": "Returns a list of all Addresses in the specified AddressSpace",
+                "operationId": "getAddressList",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "addressSpace",
+                        "in": "path",
+                        "description": "Name of AddressSpace",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/AddressList"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "addresses"
+                ],
+                "summary": "Append Addresses to an AddressSpace",
+                "description": "Append one or more Addresses to the specified AddressSpace.",
+                "operationId": "appendAddresses",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "addressSpace",
+                        "in": "path",
+                        "description": "Name of AddressSpace",
+                        "required": true,
+                        "type": "string"
+                    },
```

Comment:
does this mean you post to /addresses/addresspace/{name}?  If so I would say this is the same thing as the PUT operation below.